### ISSUE TITLE
Url returned from upload function is incorrect

### ISSUE THREAD

**Issue by Aliiiu** [ryanto/next-s3-upload#99]
After following every step in the documentation i tried to upload a file to the s3 bucket. The file was uploaded but the resonse url to the image is wrong. 

response-url: {"url":"https://gdl-luxury-yield-note.gdl-luxury-yield-note.s3.us-east-2.amazonaws.com/next-s3-uploads/6ee67ded-9f51-4908-8304-85db337c591b/Arrow.png"}

url-that-works: https://gdl-luxury-yield-note.s3.us-east-2.amazonaws.com/next-s3-uploads/6ee67ded-9f51-4908-8304-85db337c591b/Arrow.png

I will like to know if i am missing something in my setup

**Comment by ryanto** [comment]
Oh no, sorry about that!

We're using the AWS libraries to return the URL of the uploaded file. Do you mind sharing your your ENV variables so I can double check and make sure there's nothing funny there. Feel free to blank out the AWS secret :)

**Comment by Aliiiu** [comment]
S3_UPLOAD_KEY=AKLAS0WLEGBN1KTLAFNZ
S3_UPLOAD_BUCKET=gdl-luxury-yield-note
S3_UPLOAD_REGION=us-east-2

**Comment by ryanto** [comment]
@Rjsdeveloper Good to know! What region are you using?

**Comment by Rjsdeveloper** [comment]
> @Rjsdeveloper Good to know! What region are you using?


Thanks for replying, the region is us-east-1

**Comment by geraldinefiser** [comment]
I have recently added the npm package to a new app and I am also getting this issue where the name of the bucket is added at the beginning of the expected returned url, while I believe it is still working fine on my other app. 

Maybe there was an update with the plugin that changed this?

**Comment by ryanto** [comment]
Ok awesome, thanks for the debugging @geraldinefiser, that helps! What version of `@aws-sdk/lib-storage` does your older app have? And how about the new app? I'm guessing something in `@aws-sdk/lib-storage` or `@aws-sdk/client-s3` changed.

I think we should change next-s3-upload to derive the url based off the key+bucket+region, instead of relying on the aws-sdks to return the url. I'll try to get a fix out soon!

**Comment by geraldinefiser** [comment]
Older app has (next-s3-upload 0.2.2 ):
```
 "@aws-sdk/lib-storage": {
      "version": "3.131.0",
      "resolved": "https://registry.npmjs.org/@aws-sdk/lib-storage/-/lib-storage-3.131.0.tgz",
      ...
      }

```

"The new app  (next-s3-upload 0.2.1 )" has:
```
"aws-sdk": {
      "version": "2.1241.0",
      "resolved": "https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.1241.0.tgz",
      "requires": {
        "buffer": "4.9.2",
        "events": "1.1.1",
        "ieee754": "1.1.13",
        "jmespath": "0.16.0",
        "querystring": "0.2.0",
        "sax": "1.2.1",
        "url": "0.10.3",
        "util": "^0.12.4",
        "uuid": "8.0.0",
        "xml2js": "0.4.19"
      },
      "dependencies": {
        "uuid": {
          "version": "8.0.0",
          "resolved": "https://registry.npmjs.org/uuid/-/uuid-8.0.0.tgz",
          "integrity": "sha512-jOXGuXZAWdsTH7eZLtyXMqUb9EcWMGZNbL9YcGBJl4MH4nrxHmZJhEHvyLFrkxo+28uLb/NYRcStH48fnD0Vzw=="
        }
      }
    },

```

When I am npm installing the latest version of next s3 upload, I have on the new app (next-s3-upload 0.2.3):
```
"@aws-sdk/lib-storage": {
      "version": "3.197.0",
      "resolved": "https://registry.npmjs.org/@aws-sdk/lib-storage/-/lib-storage-3.197.0.tgz",
      "requires": {
        "@aws-sdk/middleware-endpoint": "3.197.0",
        "@aws-sdk/smithy-client": "3.197.0",
        "buffer": "5.6.0",
        "events": "3.3.0",
        "stream-browserify": "3.0.0",
        "tslib": "^2.3.1"
      }
    },
```

**Comment by ryanto** [comment]
@geraldinefiser  @Rjsdeveloper  @Aliiiu  Ok I just published next-s3-upload version `0.2.4` and that should hopefully fix this issue. If you upgrade to that version let me know how it goes.

**Comment by Aliiiu** [comment]
Alright will test it and revert before the week ends. Thank You so much.

**Comment by mjgasior** [comment]
Hey! Found this bug today on version `0.2.1`. Updated to `0.2.4` and it seems to be working properly now. Thx [ryanto](https://github.com/ryanto).

**Comment by Rjsdeveloper** [comment]
> @geraldinefiser @Rjsdeveloper @Aliiiu Ok I just published next-s3-upload version `0.2.4` and that should hopefully fix this issue. If you upgrade to that version let me know how it goes.

@ryanto Thank you very much for your dedication, I upgraded to 0.2.4 and it is working perfectly, congratulations for your great work, see you soon!
